bigquery/snippets/managedwriter/exampleproto/sample_data.proto

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The BigQuery Storage API expects protocol buffer data to be encoded in the
+// proto2 wire format. This allows it to disambiguate missing optional fields
+// from default values without the need for wrapper types.
+syntax = "proto2";
+package exampleproto;
+option go_package = "github.com/GoogleCloudPlatform/golang-samples/bigquery/managedwriter/exampleproto";
+
+
+
+
+// Define a message type representing the rows in your table.
+message SampleData {
+
+  // Define a nested message to encode STRUCT column values.
+  message SampleStruct {
+    optional int64 sub_int_col = 1;
+  }
```

Comment:
Do we want to unnest this for the purpose of exercising the normalize? Or call out that if the messages are nested like this, the normalization step is optional?

Reply:
Refactored the proto to have two distinct messages that require normalizing, and updated the comment.